features/admin.extensions.v1/configs/application.tsx

```diff
@@ -408,7 +408,7 @@ export const applicationConfig: ApplicationConfig = {
             // Enable the API authorization tab for supported templates when the api resources config is enabled.
             if (
                 apiResourceFeatureEnabled && !application?.advancedConfigurations?.fragment &&
-                legacyMode?.apiResourcesV1 &&
+                legacyMode?.apiResourcesV2 &&
```

Comment:
Shouldn't this be enable for both `apiResourcesV2` and `apiResourcesV1` ?

Reply:
Since we are using only V2 in IS we can go with this fix ATM.